qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -278,41 +269,23 @@ def __init__(self, parent: 'Model_325', name: str, inp: str) -> None:
             vals=Numbers(min_value=1, max_value=35)
         )
 
-    def decode_sensor_status(self, sum_of_codes: int) -> str:
-        """
-        The sensor status is one of the status codes, or a sum thereof. Multiple
-        status are possible as they are not necessarily mutually exclusive.
-
-        args:
-            sum_of_codes (int)
-        """
-        components = list(self.sensor_status_codes.keys())
-        codes = self._get_sum_terms(components, int(sum_of_codes))
-        return ", ".join([self.sensor_status_codes[k] for k in codes])
-
     @staticmethod
-    def _get_sum_terms(components: list, number: int):
-        """
-        Example:
-        >>> components = [0, 1, 16, 32, 64, 128]
-        >>> _get_sum_terms(components, 96)
-        >>> ...[64, 32]  # This is correct because 96=64+32
-        """
-        if number in components:
-            terms = [number]
-        else:
-            terms = []
-            comp = np.sort(components)[::-1]
-            comp = comp[comp <= number]
-
-            while len(comp):
-                c = comp[0]
-                number -= c
-                terms.append(c)
-
-                comp = comp[comp <= number]
-
-        return terms
+    def decode_sensor_status(sum_of_codes: int) -> str:
+        class Status(IntFlag):
```

Comment:
i wonder, why is it a private enum? i think it's OK to have it "public", at least outside this method so that it can be used in tests directly and can be referred to in code if needed elsewhere :)

Reply:
I made the `Status` "public" so that it can be called elsewhere (I called it in the test). However, I have to keep the `decode_sensor_status` as "staticmethod" because I want to call it directly in the test without defining an equipment.

So `def decode_sensor_status(sum_of_codes: int, status=Status) -> str:`
instead of  `def decode_sensor_status(self, sum_of_codes: int) -> str:`